content/webapp/components/IIIFItem/index.tsx

```diff
@@ -283,51 +299,69 @@ const IIIFItem: FunctionComponent<ItemProps> = ({
       (item.type === 'Audio' && !exclude.includes('Audio'))) &&
       !!item.id:
       return (
-        <Wrapper
+        <IIIFItemWrapper
           shouldShowItem={shouldShowItem}
           className="item-wrapper"
           titleOverride={titleOverride}
           canvas={canvas}
           isRestricted={isRestricted}
         >
           {audioPlayer && extendedViewer ? (
-            <AudioPlayerNew
-              isDark
-              audioFile={item.id}
-              title={(canvas.label !== '-' && canvas.label) || titleOverride}
-            />
+            isInViewer ? (
+              <AudioPlayerNew
+                isDark
+                audioFile={item.id}
+                title={getAudioVideoLabel(canvas.label, titleOverride) || ''}
+              />
+            ) : (
+              <IIIFItemAudioVideo
+                canvas={canvas}
+                i={i}
+                isRestricted={isRestricted}
+                item={item}
+                itemUrl={itemUrl}
+              />
+            )
           ) : (
             <AudioPlayer
               audioFile={item.id}
-              title={
-                (canvas.label !== '-' && canvas.label) || titleOverride || ''
-              }
+              title={getAudioVideoLabel(canvas.label, titleOverride) || ''}
             />
           )}
-        </Wrapper>
+        </IIIFItemWrapper>
       );
 
     case item.type === 'Video' && !exclude.includes('Video'):
       return (
-        <Wrapper
+        <IIIFItemWrapper
           shouldShowItem={shouldShowItem}
           className="item-wrapper"
           titleOverride={titleOverride}
           canvas={canvas}
           isRestricted={isRestricted}
         >
-          <>
-            <VideoPlayer
-              placeholderId={placeholderId}
-              video={item}
-              showDownloadOptions={true}
-            />
-            <VideoTranscript
-              supplementing={canvas.supplementing}
-              isDark={isDark}
+          {extendedViewer && !isInViewer ? (
```

Comment:
I _think_ this should be `extendedViewer && isInViewer`

Reply:
otherwise this displays the link and the download in the viewer instead of the video player

Reply:
Yes thank you!! It was because it was a `choice` and not of type `video`... will pass it to Choices as well!